src/librustc_middle/ty/print/pretty.rs

```diff
@@ -1177,8 +1177,13 @@ pub trait PrettyPrinter<'tcx>:
                         }
                         p!(write(")"));
                     }
+                    ty::Adt(def, substs) if def.variants.is_empty() => {
+                        p!(print_value_path(def.did, substs));
+                    }
```

Comment:
I don't think this should ever be reached? This sort of thing should be an evaluation error, surely?

Reply:
I don’t know about an evaluation error, but this definitely gets reached.

Reply:
we use the pretty printer from within the miri engine and the const evaluator does not validate that every local is always valid, only miri does this.